api/python/cellxgene_census/src/cellxgene_census/experimental/ml/pytorch.py

```diff
@@ -21,14 +22,17 @@
 from sklearn.preprocessing import LabelEncoder
 from somacore.query import _fast_csr
 from torch import Tensor
+from torch import distributed as dist
 from torch.utils.data import DataLoader
 from torch.utils.data.dataset import Dataset
 
 from ..._open import _build_soma_tiledb_context
 from ..util._eager_iter import _EagerIterator
 
 pytorch_logger = logging.getLogger("cellxgene_census.experimental.pytorch")
-
+if not pytorch_logger.hasHandlers():
+    pytorch_logger.addHandler(logging.StreamHandler(sys.stderr))
+    pytorch_logger.setLevel(logging.DEBUG)
```

Comment:
Note that the [logging docs](https://docs.python.org/3/howto/logging.html#configuring-logging-for-a-library) advise against this:
> Note It is strongly advised that you do not add any handlers other than [NullHandler](https://docs.python.org/3/library/logging.handlers.html#logging.NullHandler) to your library’s loggers.